soc/arm/silabs_exx32/common/soc_power.c

```diff
@@ -134,7 +134,7 @@ static const struct pm_state_info pm_min_residency[] =
 	PM_STATE_INFO_LIST_FROM_DT_CPU(DT_NODELABEL(cpu0));
 struct pm_state_info pm_state_active = {PM_STATE_ACTIVE, 0, 0, 0};
 
-struct pm_state_info *pm_policy_next_state(uint8_t cpu, int32_t ticks)
+__weak struct pm_state_info *pm_policy_next_state(uint8_t cpu, int32_t ticks)
```

Comment:
Put this code under `#ifdef CONFIG_PM_POLICY_CUSTOM`

Reply:
Thanks for looking at it. It already is under an `if` directive. See line 128.